account_chatter/views/account_account_views.xml

```diff
@@ -1,29 +1,24 @@
-<?xml version="1.0" encoding="utf-8" ?>
+<?xml version="1.0" encoding="UTF-8" ?>
 <odoo>
 
-    <record id="view_account_form_show_chatter" model="ir.ui.view">
+     <record id="view_account_form_show_chatter" model="ir.ui.view">
         <field name="name">account.account.form.inherit.account.chatter</field>
         <field name="model">account.account</field>
         <field name="inherit_id" ref="account.view_account_form" />
-        <field name="groups_id" eval="[(4, ref('account_chatter.group_show_account_chatter_notifications'))]" />
-        <field name="priority" eval="100" />
         <field name="arch" type="xml">
+            <xpath expr="//div[hasclass('oe_chatter')]" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </xpath>
             <xpath expr="//sheet" position="after">
-                <div class="oe_chatter">
-                    <field name="message_follower_ids" />
-                    <field name="activity_ids" />
-                    <field name="message_ids" />
-                </div>
+                <t groups="account_chatter.group_show_account_chatter_notifications">
```

Comment:
Does this work?

Take into account t directives only work in QWeb templates, they dont in standard views like form or tree view.

How is this element rendered? You may check by accessing Developer tools -> Fields view get, which shows you the rendered XML.